e2e-tests/admin/tests/change-password.spec.js

```diff
@@ -22,18 +19,14 @@ test.beforeAll(async () => {
 
 test(
   'Verify user can change password',
-  // TODO: check test works for each tag
   { tag: ['@ce', '@ent', '@aws', '@docker'] },
-  async ({
-    page,
-    controllerAddr,
-    adminAuthMethodId,
-    adminLoginName,
-    adminPassword,
-  }) => {
+  async ({ page, request, adminLoginName, adminPassword }) => {
     await page.goto('/');
-    let orgName;
-    let authMethodName;
+
+    let org;
+    let authMethod;
```

Comment:
Technically, we only need `org` and `authMethod` defined here since there are used in the `finally` block. `account` and `user` could just be defined within the try block?